test/Microsoft.AspNetCore.SignalR.Tests.Utils/VerifiableServerLoggedTest.cs

```diff
@@ -64,5 +55,12 @@ public override IDisposable StartVerifiableLog(out ILoggerFactory loggerFactory,
             var disposable = base.StartVerifiableLog(out loggerFactory, testName, ResolveExpectedErrorsFilter(expectedErrorsFilter));
             return new ServerLogScope(ServerFixture, loggerFactory, disposable);
         }
+
+        public void Dispose()
+        {
+            // Wait for the server to finish processing any in-progress requests
+            // Prevents server logging from a previous tests from showing up in test logs
+            Thread.Sleep(TimeSpan.FromMilliseconds(100));
```

Comment:
This is kinda gross..

Reply:
Yeah. I can't think of a better way to stop server logs from a previous test showing up in the next test. The only surefire way would be to start a new server for each unit test which I imagine would be slower than a small delay after server tests.